test/ffmpeg-qsv/encode/av1.py

```diff
@@ -75,6 +83,12 @@ def test(self, case, gop, bframes, tilecols, tilerows, bitrate, quality, fps, pr
     self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
     self.encode()
 
+  @slash.parametrize(*gen_av1_cbr_lp_parameters(spec_r2r))
+  def test_r2r(self, case, gop, bframes, tilecols, tilerows, bitrate, quality, fps, profile):
+    self.init(spec, case, gop, bframes, tilecols, tilerows, bitrate, fps, quality, profile)
```

Comment:
`spec` -> `spec_r2r`

Reply:
thanks review and catch this issue, updated